assets/src/apps/authoring/components/Flowchart/toolbar/FlowchartHeaderNav.tsx

```diff
@@ -178,8 +178,33 @@ export const FlowchartHeaderNav: React.FC<HeaderNavProps> = () => {
     dispatch(setShowScoringOverview({ show: true }));
   };
 
-  useKeyDown(handleUndo, ['KeyZ'], { ctrlKey: true });
-  useKeyDown(handleRedo, ['KeyY'], { ctrlKey: true });
+  useKeyDown(
+    (ctrlKey, metaKey, shiftKey) => {
+      if (ctrlKey && !shiftKey) {
+        handleUndo();
+      }
+    },
+    ['KeyZ'],
+    { ctrlKey: true },
+  );
+  useKeyDown(
+    (ctrlKey, metaKey) => {
+      if (ctrlKey && !metaKey) {
```

Comment:
changes for request -> On Mac, cmd+y works but it also opens chrome history, so it makes it unusable, can we remove this keyboard control for Mac?